src/main/java/interface_adapter/club_logged_in/ClubLoggedInState.java

```diff
@@ -0,0 +1,68 @@
+package interface_adapter.club_logged_in;
+
+/**
+ * Club Logged In state.
+ */
+public class ClubLoggedInState {
+    private String clubName = "";
+
+    private String email = "";
+    private String emailError;
+
+    private String password = "";
+    private String passwordError;
```

Comment:
Error instance variables for email and password are not needed for this state. There are no cases in the club home view where an error can be made from these values